### PR TITLE
Making RSS valid and displaying the valid badge

### PR DESCRIPTION
**Pull request by procrastinatio**
Notes:
* Normalized on https
* Using staging and branch names

[rss v2](https://mf-chsdi3.dev.bgdi.ch/mom_fix_rss/releasenotes/rss2.xml)

[release note page](https://mf-chsdi3.dev.bgdi.ch/mom_fix_rss/releasenotes/)

[validator](https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fmf-chsdi3.dev.bgdi.ch%2Fmom_fix_rss%2Freleasenotes%2Frss2.xml)

See https://github.com/geoadmin/mf-chsdi3/issues/2557

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -156,7 +156,7 @@ doc: chsdi/static/css/extended.min.css
 .PHONY:
 rss: doc chsdi/static/doc/build/releasenotes/index.html
 	@echo "${GREEN}Creating the rss feed from releasenotes${RESET}";
-	${PYTHON_CMD} scripts/rssFeedGen.py
+	${PYTHON_CMD} scripts/rssFeedGen.py "https:${API_URL}"
```
Comment (loicgasser) [inline Makefile]: Let's not publish dev/int links in RSS Feed. Always use prod URL, it's safer.

**Comment by procrastinatio** [comment]
@loicgasser As you want, but using only api3.geo.admin.ch means you can only validate on prod.

**Comment by procrastinatio** [comment]
jenkins tests

**Comment by procrastinatio** [comment]
jenkins test

**Comment by loicgasser** [comment]
yes It should be fine now that you pimped it a bit. Just really don't want to end up with rss feed and dev/int URLs.

**Comment by procrastinatio** [comment]
So now we care about not spreading dev and int URLs?

**Pull request merged**